Kinvey.Core/Auth/User.cs

```diff
@@ -549,7 +549,7 @@ static public void LoginWithMIC(string redirectURI, KinveyMICDelegate<User> MICD
 
 			ct.ThrowIfCancellationRequested();
 
-			string myURLToRender = hostname + "oauth/auth?client_id=" + clientID + "&redirect_uri=" + redirectURI + "&response_type=code";
+			string myURLToRender = hostname + "oauth/auth?client_id=" + clientID + "&redirect_uri=" + redirectURI + "&response_type=code" + "&scope=openid";
```

Comment:
this + `kinveyApiVersion = "4"` should be the real change right? if so, the PR should be touching a smaller number of files